.gitignore

```diff
@@ -7,8 +7,3 @@ __pycache__
 htmlcov
 models
```

Comment:
juste realizing that this folder is also created as part of the build/training. This declaration should be moved inside `/core` and be more specific (`/models`) to avoid ignoring _all_ folders legitimately called models.